docs/data/base/components/slider/slider.md

```diff
@@ -174,6 +174,17 @@ To let users set the start and end of a range on a slider, provide an array of v
 
 {{"demo": "RangeSlider.js"}}
 
+### Value label
+
+The value label can be rendered over the thumb by using the optional `slots` prop with the `valueLabel` slot. The value label can either be:
+
+- Always shown by providing the `slots.valueLabel`.
+- Show only when it's hovered over the thumb (using CSS).
```

Comment:
```suggestion
- always 
- only when hovering over the thumb - using CSS
```

We already mentioned anove that people need to use the `valueLabel` slot :) Additionally, even if you are using CSS in the second step, the slot still needs to be passed